src/framework/components/element/system.js

```diff
@@ -76,10 +76,7 @@ class ElementComponentSystem extends ComponentSystem {
         this.defaultScreenSpaceImageMaskMaterial = null;
 
         // text element materials created on demand by getTextElementMaterial()
-        this.defaultTextMaterial = null;
-        this.defaultBitmapTextMaterial = null;
-        this.defaultScreenSpaceTextMaterial = null;
-        this.defaultScreenSpaceBitmapTextMaterial = null;
+        this._defaultMaterials = {};
```

Comment:
These materials are just for text, right? If so, I think it is best to rename to `_defaultTextMaterials`

Reply:
Fixed